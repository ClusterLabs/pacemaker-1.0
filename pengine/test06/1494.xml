<cib admin_epoch="0" have_quorum="true" num_peers="2" cib_feature_revision="1.3" ignore_dtd="false" generated="true" epoch="537" num_updates="22245" cib-last-written="Mon Feb 19 13:49:24 2007" ccm_transition="2" dc_uuid="1ce670eb-d670-4f6d-811f-d415ffde76b7">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options">
         <attributes>
           <nvpair name="last-lrm-refresh" id="cib-bootstrap-options-last-lrm-refresh" value="1171856040"/>
           <nvpair id="cib-bootstrap-options-default_resource_stickiness" name="default_resource_stickiness" value="INFINITY"/>
           <nvpair name="stonith_enabled" id="cib-bootstrap-options-stonith_enabled" value="False"/>
           <nvpair id="cib-bootstrap-options-no_quorum_policy" name="no_quorum_policy" value="ignore"/>
         </attributes>
       </cluster_property_set>
     </crm_config>
     <nodes>
       <node id="af77f4a5-5626-43b3-acbe-9299ae520262" uname="hb3" type="normal">
         <instance_attributes id="nodes-af77f4a5-5626-43b3-acbe-9299ae520262">
           <attributes>
             <nvpair id="standby-af77f4a5-5626-43b3-acbe-9299ae520262" name="standby" value="off"/>
           </attributes>
         </instance_attributes>
         <instance_attributes id="master-af77f4a5-5626-43b3-acbe-9299ae520262">
           <attributes/>
         </instance_attributes>
       </node>
       <node uname="hb2" type="normal" id="86ad895f-1387-461e-a9b5-c52ed9537e13">
         <instance_attributes id="nodes-86ad895f-1387-461e-a9b5-c52ed9537e13">
           <attributes>
             <nvpair id="standby-86ad895f-1387-461e-a9b5-c52ed9537e13" name="standby" value="off"/>
             <nvpair id="nodes-86ad895f-1387-461e-a9b5-c52ed9537e13-inhibit_group0" name="inhibit_group0" value="1"/>
           </attributes>
         </instance_attributes>
         <instance_attributes id="master-86ad895f-1387-461e-a9b5-c52ed9537e13">
           <attributes>
             <nvpair id="master-resource_masterslave:0-86ad895f-1387-461e-a9b5-c52ed9537e13" name="master-resource_masterslave:0" value="1"/>
           </attributes>
         </instance_attributes>
       </node>
       <node id="1ce670eb-d670-4f6d-811f-d415ffde76b7" uname="hb1" type="normal">
         <instance_attributes id="nodes-1ce670eb-d670-4f6d-811f-d415ffde76b7">
           <attributes>
             <nvpair id="standby-1ce670eb-d670-4f6d-811f-d415ffde76b7" name="standby" value="off"/>
             <nvpair id="nodes-1ce670eb-d670-4f6d-811f-d415ffde76b7-db_in_sync" name="db_in_sync" value="1"/>
           </attributes>
         </instance_attributes>
         <instance_attributes id="master-1ce670eb-d670-4f6d-811f-d415ffde76b7">
           <attributes/>
         </instance_attributes>
       </node>
     </nodes>
     <resources>
       <clone id="ima_cloneid">
         <instance_attributes id="ima_cloneid_instance_attrs">
           <attributes>
             <nvpair id="ima_cloneid_clone_max" name="clone_max" value="2"/>
             <nvpair id="ima_cloneid_clone_node_max" name="clone_node_max" value="1"/>
           </attributes>
         </instance_attributes>
         <primitive class="ocf" type="Dummy" provider="heartbeat" id="ima_rscid">
           <instance_attributes id="ima_rscid_instance_attrs">
             <attributes>
               <nvpair name="target_role" id="ima_rscid_target_role" value="started"/>
             </attributes>
           </instance_attributes>
         </primitive>
       </clone>
     </resources>
     <constraints>
       <rsc_location id="place_ima_cloneid" rsc="ima_cloneid">
         <rule id="preferred_place_ima_cloneid" score="-INFINITY">
           <expression attribute="#uname" id="c4e38253-a1cd-4aaa-aaca-f717c3a51e1d" operation="eq" value="hb1"/>
         </rule>
       </rsc_location>
     </constraints>
   </configuration>
   <status>
     <node_state id="1ce670eb-d670-4f6d-811f-d415ffde76b7" uname="hb1" crmd="online" crm-debug-origin="do_update_resource" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <transient_attributes id="1ce670eb-d670-4f6d-811f-d415ffde76b7">
         <instance_attributes id="status-1ce670eb-d670-4f6d-811f-d415ffde76b7">
           <attributes>
             <nvpair id="status-1ce670eb-d670-4f6d-811f-d415ffde76b7-probe_complete" name="probe_complete" value="true"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
       <lrm id="1ce670eb-d670-4f6d-811f-d415ffde76b7">
         <lrm_resources>
           <lrm_resource id="ima_rscid:0" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid:0_monitor_0" operation="monitor" crm-debug-origin="build_active_RAs" transition_key="3:3:4bebe060-ef49-4030-9528-60e8e57310bd" transition_magic="4:7;3:3:4bebe060-ef49-4030-9528-60e8e57310bd" call_id="12" crm_feature_set="1.0.5" rc_code="7" op_status="4" interval="0" op_digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
             <lrm_rsc_op id="ima_rscid:0_start_0" operation="start" crm-debug-origin="build_active_RAs" transition_key="16:5:4bebe060-ef49-4030-9528-60e8e57310bd" transition_magic="0:0;16:5:4bebe060-ef49-4030-9528-60e8e57310bd" call_id="14" crm_feature_set="1.0.5" rc_code="0" op_status="0" interval="0" op_digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
           <lrm_resource id="ima_rscid:1" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid:1_monitor_0" operation="monitor" crm-debug-origin="build_active_RAs" transition_key="12:0:4bebe060-ef49-4030-9528-60e8e57310bd" transition_magic="4:7;12:0:4bebe060-ef49-4030-9528-60e8e57310bd" call_id="11" crm_feature_set="1.0.5" rc_code="7" op_status="4" interval="0" op_digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
     </node_state>
     <node_state id="86ad895f-1387-461e-a9b5-c52ed9537e13" uname="hb2" ha="active" crm-debug-origin="do_update_resource" crmd="online" shutdown="0" in_ccm="true" join="member" expected="member">
       <lrm id="86ad895f-1387-461e-a9b5-c52ed9537e13">
         <lrm_resources>
           <lrm_resource id="ima_rscid:0" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid:0_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="11:6:4bebe060-ef49-4030-9528-60e8e57310bd" transition_magic="0:7;11:6:4bebe060-ef49-4030-9528-60e8e57310bd" call_id="10" crm_feature_set="1.0.5" rc_code="7" op_status="0" interval="0" op_digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
           <lrm_resource id="ima_rscid:1" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid:1_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="12:6:4bebe060-ef49-4030-9528-60e8e57310bd" transition_magic="0:7;12:6:4bebe060-ef49-4030-9528-60e8e57310bd" call_id="11" crm_feature_set="1.0.5" rc_code="7" op_status="0" interval="0" op_digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
             <lrm_rsc_op id="ima_rscid:1_start_0" operation="start" crm-debug-origin="do_update_resource" transition_key="30:6:4bebe060-ef49-4030-9528-60e8e57310bd" transition_magic="0:0;30:6:4bebe060-ef49-4030-9528-60e8e57310bd" call_id="12" crm_feature_set="1.0.5" rc_code="0" op_status="0" interval="0" op_digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="86ad895f-1387-461e-a9b5-c52ed9537e13">
         <instance_attributes id="status-86ad895f-1387-461e-a9b5-c52ed9537e13">
           <attributes>
             <nvpair id="status-86ad895f-1387-461e-a9b5-c52ed9537e13-probe_complete" name="probe_complete" value="true"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
     </node_state>
   </status>
 </cib>
