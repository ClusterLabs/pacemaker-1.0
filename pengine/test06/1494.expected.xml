<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="537" num_updates="22245" dc-uuid="1ce670eb-d670-4f6d-811f-d415ffde76b7" have-quorum="true" remote-tls-port="0" validate-with="pacemaker-1.0">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options"><nvpair name="last-lrm-refresh" id="nvpair" value="1171856040"/><nvpair id="nvpair" name="default-resource-stickiness" value="INFINITY"/><nvpair name="stonith-enabled" id="nvpair" value="False"/><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/></cluster_property_set>
     </crm_config>
     <nodes>
       <node id="af77f4a5-5626-43b3-acbe-9299ae520262" uname="hb3" type="normal">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="standby" value="off"/></instance_attributes>
         <instance_attributes id="instance_attributes"/>
       </node>
       <node id="86ad895f-1387-461e-a9b5-c52ed9537e13" uname="hb2" type="normal">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="standby" value="off"/><nvpair id="nvpair" name="inhibit_group0" value="1"/></instance_attributes>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="master-resource_masterslave:0" value="1"/></instance_attributes>
       </node>
       <node id="1ce670eb-d670-4f6d-811f-d415ffde76b7" uname="hb1" type="normal">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="standby" value="off"/><nvpair id="nvpair" name="db_in_sync" value="1"/></instance_attributes>
         <instance_attributes id="instance_attributes"/>
       </node>
     </nodes>
     <resources>
       <clone id="ima_cloneid"><meta_attributes id="clone-ima_cloneid.meta"><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/></meta_attributes><primitive id="ima_rscid" class="ocf" type="Dummy" provider="heartbeat"><meta_attributes id="primitive-ima_rscid.meta"><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes>
           <instance_attributes id="instance_attributes"><nvpair name="target_role" id="nvpair" value="started"/></instance_attributes>
         </primitive></clone>
     </resources>
     <constraints>
       <rsc_location id="place_ima_cloneid" rsc="ima_cloneid">
         <rule id="preferred_place_ima_cloneid" score="-INFINITY">
           <expression attribute="#uname" id="expression" operation="eq" value="hb1"/>
         </rule>
       </rsc_location>
     </constraints>
   </configuration>
   <status>
     <node_state id="1ce670eb-d670-4f6d-811f-d415ffde76b7" uname="hb1" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <transient_attributes id="transient_attributes.auto-1">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
       </transient_attributes>
       <lrm id="lrm.auto-1">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="ima_rscid:0" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid-0_monitor_0" operation="monitor" transition-key="3:3:4bebe060-ef49-4030-9528-60e8e57310bd" transition-magic="4:7;3:3:4bebe060-ef49-4030-9528-60e8e57310bd" call-id="12" crm_feature_set="1.0.5" rc-code="7" op-status="4" interval="0" op-digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
             <lrm_rsc_op id="ima_rscid-0_start_0" operation="start" transition-key="16:5:4bebe060-ef49-4030-9528-60e8e57310bd" transition-magic="0:0;16:5:4bebe060-ef49-4030-9528-60e8e57310bd" call-id="14" crm_feature_set="1.0.5" rc-code="0" op-status="0" interval="0" op-digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
           <lrm_resource id="ima_rscid:1" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid-1_monitor_0" operation="monitor" transition-key="12:0:4bebe060-ef49-4030-9528-60e8e57310bd" transition-magic="4:7;12:0:4bebe060-ef49-4030-9528-60e8e57310bd" call-id="11" crm_feature_set="1.0.5" rc-code="7" op-status="4" interval="0" op-digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
     </node_state>
     <node_state id="86ad895f-1387-461e-a9b5-c52ed9537e13" uname="hb2" ha="active" crmd="online" shutdown="0" in_ccm="true" join="member" expected="member">
       <lrm id="lrm.auto-2">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="ima_rscid:0" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid-0_monitor_0" operation="monitor" transition-key="11:6:4bebe060-ef49-4030-9528-60e8e57310bd" transition-magic="0:7;11:6:4bebe060-ef49-4030-9528-60e8e57310bd" call-id="10" crm_feature_set="1.0.5" rc-code="7" op-status="0" interval="0" op-digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
           <lrm_resource id="ima_rscid:1" type="Dummy" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="ima_rscid-1_monitor_0" operation="monitor" transition-key="12:6:4bebe060-ef49-4030-9528-60e8e57310bd" transition-magic="0:7;12:6:4bebe060-ef49-4030-9528-60e8e57310bd" call-id="11" crm_feature_set="1.0.5" rc-code="7" op-status="0" interval="0" op-digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
             <lrm_rsc_op id="ima_rscid-1_start_0" operation="start" transition-key="30:6:4bebe060-ef49-4030-9528-60e8e57310bd" transition-magic="0:0;30:6:4bebe060-ef49-4030-9528-60e8e57310bd" call-id="12" crm_feature_set="1.0.5" rc-code="0" op-status="0" interval="0" op-digest="08b7001b97ccdaa1ca23a9f165256bc1"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="transient_attributes.auto-2">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
       </transient_attributes>
     </node_state>
   </status>
 </cib>
